docs/ecs_guide.md

```diff
@@ -0,0 +1,329 @@
+3. **AWS Integration**: Easily connect with other AWS services, such as AWS IAM and CloudWatch.
+4. **Containerization**: ECS supports Docker containers and offers managed execution. Containerization encourages reproducable deployments.
+
+## ECS Flow Run Execution
+
+To set up Prefect flow execution in your cloud infrastructure, you need to understand two key Prefect concepts: 
+
+- a [worker](https://docs.prefect.io/2.10.11/concepts/work-pools/#worker-overview) to poll Prefect's API (specifically that worker's [work queue](https://docs.prefect.io/2.10.11/concepts/work-pools/#work-queues)) 
+- and the emphemeral [infrastructure](https://docs.prefect.io/2.10.11/concepts/infrastructure/) that will be spun up by this worker for each scheduled [flow run](https://docs.prefect.io/2.10.11/concepts/flows/#flow-runs). 
+
+#### Archetecture Diagram
+```mermaid
+graph TB
+
+
+  subgraph ecs_cluster[ECS cluster]
+    subgraph ecs_service[ECS service]
+      td_worker[worker task definition] --> |restarts| prefect_worker((Prefect worker))
+    end
+    prefect_worker -->|kicks off| ecs_task
+    fr_task_definition[flow run task definition]
+
+
+    subgraph ecs_task["ECS Task Execution <br>  (flow run infrastructure)"]
+    style ecs_task text-align:center
+      
+      flow_run((Flow Run))
+
+    end
+    fr_task_definition -->|blueprint for| ecs_task
+  end
+
+  subgraph prefect_cloud[Prefect Cloud]
+    subgraph prefect_workpool[ECS work pool]
+      default_workqueue[Default work queue]
+    end
+  end
+
+  flow_code["Flow Code (e.g. GitHub)"] --> |prefect project pull step| ecs_task
+  prefect_worker -->|polls| default_workqueue
+  prefect_workpool -->|configures| fr_task_definition
+```
+### ECS in Prefect Terms
+!!! tip "ECS tasks != Prefect tasks"
+    An ECS Task is **not** the same thing as a [Prefect task](https://docs.prefect.io/2.10.11/concepts/tasks/#tasks-overview). 
+    
+    ECS tasks are run as part of an ECS Cluster, they launch containers as defined in the ECS Task definition. 
+    
+    An [*ECS task definition*](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) is the blueprint for the ECS task. It describes which Docker containers to run and what you want to have happen inside these containers.
+
+ECS Tasks are instances of a task definition. A Task Execution launches container(s) as defined in the task definition **until they are stopped or exit on their own**. This setup is ideal for ephemeral processes such as a Prefect flow run.
+
+The ECS task running the Prefect [worker](https://docs.prefect.io/2.10.11/concepts/work-pools/#worker-overview) should be an [**ECS Service**](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html), given its long-running nature and need for **auto-recovery in case of failure**. An ECS service automatically replaces any task that fails, which is ideal for managing a long-running process such as a Prefect worker.
```

Comment:
```suggestion
The ECS task running the Prefect [worker](https://docs.prefect.io/latest/concepts/work-pools/#worker-overview) should be an [**ECS Service**](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html), given its long-running nature and need for **auto-recovery in case of failure**. An ECS service automatically replaces any task that fails, which is ideal for managing a long-running process such as a Prefect worker.
```